scania-digital-design-system/sdds
# fix(badge): replace size prop value default with large

**Describe pull-request**  
Replace size prop value default with large

**Solving issue**  
Fixes: [AB#2649](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2649)

**How to test**  
1. Go to Netlify preview link
2. Check Badge component
3. See if size values are Large("lg") and Small("sm") in Canvas, Docs and in Migration guide
